README.md

```diff
@@ -1,103 +1,151 @@
- - `ntp-daemon` contains the main NTP daemon, and deals with orchestrating the networking and configuration.
- - `ntp-ctl` contains a control interface for the NTP daemon, allowing readout of current synchronisation state and dynamic configuration changes.
- - `ntp-metrics-exporter` contains a HTTP interface for exporting the prometheus metrics.
- - `test-binaries` contains a number of simple NTP servers that can be used for testing (see below).
- - `ntp-os-clock` contains the unsafe code needed to interface with system clocks.
- - `ntp-udp` contains the unsafe code needed to deal with timestamping on the network layer.
- - `ntpd` contains the entrypoints for all our binaries
 
-All unsafe code is contained within the `ntp-os-clock` and `ntp-udp` packages, which are kept as small as possible. All interfaces exposed by these crates should be safe. For a more detailed description of how ntpd-rs is structured, see the [development documentation](DEVELOPMENT.md).
+* `ntp-proto` contains the packet parsing and the algorithms needed for clock
+  selection, filtering and steering.
+* `test-binaries` contains a number of simple NTP servers that can be used for
+  testing (see below).
+* `ntp-os-clock` contains the unsafe code needed to interface with system
+  clocks.
+* `ntp-udp` contains the unsafe code needed to deal with timestamping on the
+  network layer.
+* `ntpd` contains the entrypoints for all our binaries and the code for the
+  daemon (`ntp-daemon`), control client (`ntp-ctl`) and OpenMetrics/prometheus
+  exporter (`ntp-metrics-exporter`).
+
+
+All unsafe code is contained within the `ntp-os-clock` and `ntp-udp` packages,
+which are kept as small as possible. All interfaces exposed by these crates
+should be safe. For a more detailed description of how ntpd-rs is structured,
+see the [development documentation](./docs/development/).
 
 ## Test Binaries
 
 This crate contains extremely limited NTP servers for testing purposes
 
-* `demobilize-server` always sends the DENY kiss code, the client must demobilize this association
+* `demobilize-server` always sends the DENY kiss code, the client must
+  demobilize this association
 * `rate-limit-server` forces an increase of the poll interval to 32 seconds
 
 ## Roadmap
 
-In Q1 2023 we completed our work on NTS. Our implementation is now full-featured, it supports NTP client and server with NTS.
+In Q1 2023 we completed our work on NTS. Our implementation is now
+full-featured, it supports NTP client and server with NTS.
 
 Our roadmap for 2023:
 
-- Q2 2023: Adoption work, improved packaging, FreeBSD support
-- Q3 2023: Stable release (pending funding)
-- Q4 2023: Development work on experimental features, NTS pool, NTPv5 (pending funding)
+* Q2 2023: Adoption work, improved packaging, FreeBSD support
+* Q3 2023: Stable release (pending funding)
+* Q4 2023: Development work on experimental features, NTS pool, NTPv5 (pending
```

Comment:
this funding has now been secured, right?

Reply:
I think so, but I'd rather let someone else do the content updates here.